Lib/logging/handlers.py

```diff
@@ -196,9 +196,12 @@ def shouldRollover(self, record):
         if self.stream is None:                 # delay was set...
             self.stream = self._open()
         if self.maxBytes > 0:                   # are we rolling over?
+            pos = self.stream.tell()
+            if not pos:
+                # gh-116263: Never rollover an empty file
+                return False
             msg = "%s\n" % self.format(record)
-            self.stream.seek(0, 2)  #due to non-posix-compliant Windows feature
-            if self.stream.tell() + len(msg) >= self.maxBytes:
+            if pos + len(msg) >= self.maxBytes:
```

Comment:
The preceding line hardcodes the `\n` terminator - in the PR for gh-116267, I've changed it to avoid that. same comment as above re. clash with the PR for gh-116267.

Reply:
Yes, but it will be not difficult to resolve conflict.